backend/apps/core/apps.py

```
@@ -1,9 +1,38 @@
+        ).strip().lower()
+
+        api_key = str(
+            getattr(settings, "ALGOLIA_WRITE_API_KEY", "")
+        ).strip().lower()
+
+        if app_id in invalid_values or api_key in invalid_values:
+            logger.warning(
+                "Algolia is not configured. "
+                "Please set ALGOLIA_APPLICATION_ID and "
+                "ALGOLIA_WRITE_API_KEY in environment variables."
+            )
```

Ruff (0.15.14)

[warning] No newline at end of file

Add trailing newline

(W292)